### PR TITLE
[WIP] Report page

### PR DESCRIPTION
**Pull request by DavidEBest**
Very rough still.

Things to do: 
- [x] Prettify the date ('Today at ', etc.) in javascript or Ruby
- [x] Sortable table by Type & Risk
- [ ] General styling fixes
- [x] Add the json download link (might need to change the ruby endpoint?)

Wireframe
cv-wireframes-v3_page_3

### CODE REVIEW

**Comment by jseppi** [comment]
- Prettified the updated_at time with moment.js
- Made the risk table sortable with stupidtable (jquery plugin)
- Added `assets/javascripts/vendor/` folder with jquery, moment.js, and stupidtable

The table styling is wonky. I believe the `cloudgov-style.css` needs to be updated because it seems to have the same issue alluded to in https://github.com/18F/cg-style/pull/51

**Comment by msecret** [comment]
Yea @jseppi we resolve this is v0.4.0, I suggest updating as that update _shouldn't_ have any breaking changes.

Speaking of which, it looks like you guys just ended up copying the cg-style files and assets into the project and committed them. Do you have a plan for updating cg-style? Or if you want to do active development on cg-style while working on compliance-viewer?

File: assets/javascripts/app.js [hunk 1]
```diff
@@ -1,3 +1,19 @@
 // assets/javascripts/app.js
 
-//= require_tree .
\ No newline at end of file
+//= require_tree .
+
+(function($) {
+
+  //Format <time> elements
+  $('.time-human-readable').each(function (){
```
Comment (msecret) [inline assets/javascripts/app.js]: A pattern in the front end guide is to prefix classes used as js hooks with `js-`, so `js-time-human-readable`.

**Comment by msecret** [comment]
If you're creating any new design elements, please let me know or contribute them back up to `cloudgov-style` (as long as their not compliance-viewer specific.

**Comment by DavidEBest** [comment]
@jseppi Great! If you have the time, feel free to upgrade to cg-style 0.4.0.

**Comment by jseppi** [comment]
Updated cg-style (locally). It seems like it would be best to use the cg-style-gem (https://github.com/18F/cg-style-gem) but I don't know enough about ruby's asset pipeline to get that working currently.

**Comment by DavidEBest** [comment]
Yeah, we talked about that a few days ago here: https://github.com/18F/compliance-viewer/pull/21#issuecomment-212559046. That seems like it'd be a worthwhile thing to tackle, especially if you are still looking for tasks. @msecret

**Comment by jseppi** [comment]
Sure, I can look into it. I just might be slow!

**Comment by jseppi** [comment]
Attempt at using cloudgov-style from npm implemented in #30

**Comment by DavidEBest** [comment]
Type and Risk are sortable, there should be a visual indicator of that capability.

screenshot 2016-05-02 11 52 58

**Comment by juliasolorzano** [comment]
Nice work! There are a few things to update:
- [ ] Remove background image and style similar to https://cloud.gov/updates/
- [ ] Add footer cloud.gov styling
- [ ] Add back arrow to "Project List page" link at the top of the page
- [ ] Add spacing (see wireframe) for the top portion of the page (see image below).
- [ ] Make "Updated..." italicized
- [ ] Add relevant links to each bullet point under "Getting Started"

Please let me know if you'd like to go over anything. I am also happy to add these fixes directly to the branch too.

screen shot 2016-05-02 at 12 06 36 pm

**Comment by juliasolorzano** [comment]
Also, just added the wireframe to the description for reference 😄

**Comment by afeld** [comment]
One suggestion: since this branch only contains improvements and no breakage, I'd suggest merging it sooner than later so that we can follow up with smaller pull requests. I defer to y'all when that good tipping point would be.

**Comment by DavidEBest** [comment]
Good call, @afeld. @jseppi - if you are okay with it, I'll merge this now, and we can move the remaining things into a few parallelizable streams of work.

**Comment by jseppi** [comment]
Works for me

**Pull request merged**